pkg/controllers/mcs/service_export_controller.go

```diff
@@ -64,6 +64,9 @@ import (
 const ServiceExportControllerName = "service-export-controller"
 
 // ServiceExportController is to sync ServiceExport and report EndpointSlices of exported service to control-plane.
+// In order to avoid creating informer for each member cluster, the controller will watch Work resources containing
```

Comment:
misleading. The controller needs to create informer for member clusters, but only for clusters where have the ServiceExport resource propagated.

Reply:
Updated, can you help review it again?

Reply:
OK.